salt/loader.py

```diff
@@ -657,7 +655,12 @@ def render(opts, functions, states=None, proxy=None, context=None):
 
     if states:
         pack["__states__"] = states
-    pack["__proxy__"] = proxy or {}
+
+    if proxy is None:
+        pack["__proxy__"] = {}
+    else:
+        pack["proxy"] = proxy
```

Comment:
Not `pack["__proxy__"]`?

And if that's the case why not
```python
If proxy is None:
    proxy = {}
pack["__proxy__"] = proxy
```

Reply:
Fixed